packages/components/modules/profiles/ProfilePopover/SwitchProfileMenu/TestTwo.tsx

```diff
@@ -0,0 +1,8 @@
+const Test = () => (
```

Comment:
I do not get what this does or why it is there. (Might be me being unexperienced, though.)